### DISCUSSION TITLE
关于金融工作会议的解读

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#17] [General]
有网友预测到了以后的事情

Original 小晖  战壕GPS  2023-11-05 15:48

**Comment by Lewisrxliu** [comment]
ZT 都说风起于青苹之末，聪明的人只需要看到一点小小的动向，就知道接下来会发生什么事，这样的网友可以算是上是智者了。而就在这两天，有智者型网友通过一个会议，看到了以后会发生的事，言语间很是悲凉，于是我又想写出来了。前两天开了一个会，这个网友认真去看了，看完以后有所心得，于是就分享出来了。这是他的第一个解读，说以后市场规律不好使了。第二个心得是，未来市场的作用会进一步被削弱。第三个心得是，以后地方债也不会怎么管了，爱爆不爆。第四个心得是，以后中小银行基本上没有太大的发展空间了。第五个心得是，以后地产加杠杆的行为不被允许了，于是房地产也不会像以前那么火热了，地产时代可以宣告结束。第六个心得是，以后不能大面积、大规模的暴雷了，谁暴雷谁扛，但是否要暴雷，还要根据大局来走。第七个要点是，未来一面时间，货币超发行为将持续存在，同时跨周节和逆周节，也可以看出不再把市场周期和经济周期当回事了。第八个心得是，以后保险业的理财产品不能买了。第九个要点是，以后学金融的最好去上海，其他城市基本上不会有太多机会。总结起来，就是以后经济这一块，不再是市场主导，也不再是经济规律主导，更不是专业人士主导，而是由行政机构的意志主导。这就是网友看到的未来时代面貌，我也想让大家看到，然后记在心里，看以后是否会一一验证。就这样。

**Reply by Lewisrxliu** [reply]
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/fe9faa05-38b4-4ba8-b6de-c4b8e2505958)

**Comment by Lewisrxliu** [comment]
http://www.china-cer.com.cn/guwen/2023110126406.html

**Comment by Lewisrxliu** [comment]
https://www.cbirc.gov.cn/cn/view/pages/ItemDetail.html?docId=1134368&itemId=915

**Comment by Lewisrxliu** [comment]
https://wallstreetcn.com/articles/3700969

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/4ae10b71-e6a9-43a5-9e4f-e68d58d04f8a)

**Comment by ergatea** [comment]
https://www.xiaoyuzhoufm.com/episode/6548c9cf1040d2820c083db1

**Reply by Lewisrxliu** [reply]
最后这个莎士比亚的故事，令人印象深刻

**Reply by ergatea** [reply]
关于未来我们一无所知

**Reply by guttasingapore** [reply]
未来无限可能性：第10讲同学讨论关键词： 恶性通胀，无锚印钞，抛售外债，外贸下跌，票证经济，外源断裂，。。。。 各位悲观主义的同学们，能不能向XP同学学习一下不可救药的乐观主义精神啊！

**Comment by guttasingapore** [comment]
今日读报2023-11.28
中国人民银行等八部门联合印发《关于强化金融支持举措 助力民营经济发展壮大的通知》
http://www.news.cn/2023-11/27/c_1129995048.htm
http://www.news.cn/fortune/2023-11/27/c_1129995103.htm
https://wb.beijing.gov.cn/home/index/rdgz/202311/t20231128_3312030.html
https://www.cna.com.tw/news/acn/202311270095.aspx
https://www.stcn.com/article/detail/998726.html

**Comment by guttasingapore** [comment]
Monetary Authority of Singapore (MAS) 新加坡金融管理局2023年度金融稳定评估报告
https://www.mas.gov.sg/publications/financial-stability-review/2023/financial-stability-review-2023

**Comment by Lewisrxliu** [comment]
关于中国的人口和生育问题
假设观点能够以数据支撑
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/a31ab7f8-fa14-43d1-a7f3-311fea87826b)

**Reply by Lewisrxliu** [reply]
党领导外事工作条例：涉外法制建设意味着什么
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/1de3dc19-2226-4279-ae59-e0c17597809d)![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/2a44b09d-7c96-4aaf-8ccd-a12669855df7)

**Comment by ergatea** [comment]
The current incentives embedded in the inter-government fiscal system in China have allowed local
debt to accumulate rapidly and for repayment pressure to escalate. Future policies need both to
address the existing debt amount and repayment pressure as well as to create incentives to slow
down the creation of new debt. On dealing with existing debt, the most obvious policy is to smooth
out the repayment schedule of local debt. To create incentives to slow down the pace of debt creation,
China already relies on the cadre responsibility system to exert political pressure on local leaders
(Ministry of Finance, 2016). Yet, local debt continues to accumulate at an alarming rate. The reason
is that central policy demands in other areas continue to multiply, which require greater budgetary
outlays at the local level. This in turn creates room for opportunistic spending at the local level. The
centralization of budgetary outlays would not only take discretion out of the hands of local officials,
but also immediately make central deficit spending apparent, thus potentially imposing some degree
of fiscal discipline at the central level. Currently, the central government can enact unfunded mandates
without thinking much of the budgetary consequences, implicitly pressuring local authorities to borrow
to provide for these mandates. Because of the scale of the problem, any fundamental resolution of the
current debt dynamic will be difficult both economically and politically.
[Uploading 2023-report_shih_local-government-debt-dynamics-in-china.pdf…]()

**Reply by Lewisrxliu** [reply]
👍 
加强学习， 转帖一篇 陆家嘴金融  信贷白话  2023-12-03
里程碑式的会议：对金融工作的评价和指导

转自：陆家嘴金融1.本次会议被称作里程碑式的会议，对过去的评价不高，负面。金融被视为血脉，而非核心。2.本次会议最重要的改革是党管金融体制，组建中央金融委员会。该委员会将管理政策、人事和股权。各省将成立金融委员会，国务院金融稳定发展委员会及其办事机构将被取消。一些重要岗位将不再需要留学经历。3.党管金融的核心理论在于，信用债是党的信用。最后是政治局兜底，以防止党的信用被透支。4.党管金融的政治性问题不容忽视，因此厅局级以上领导干部的配偶等不允许从事私募股权基金投资及从业等，以防止出现“华尔街政治”。5.一行两会的权威不足，党将管理干部，并关注金融腐败。6.目前面临的问题不是降薪，而是追薪。已经发放的资金还需要追回。7.金融监管总局将成为一个超级监管机构，强化机构监管、行为监管、功能监管、穿透式监管和持续监管。金监总局将进行全覆盖，不再是谁发牌照谁监管。8.金融监局下将设立稽查局，与执法机关合作，并参照证监会的稽查队伍设置。9.中央支持货币政策和金融监管可以分离的理论。10.这次会议后，处置小银行不良资产等工作明显加快。11.中国不能盲目追求美国金融模式，而应走中国特色金融发展道路。金融应作为服务业，为实体经济和高科技服务。不应鼓励把金融业发展作为经济发展的优先地位。地方经济也遵循同样的原则。许多地方已经不再使用金融科技的概念。中国不需要太多的金融中心城市，如上海和香港。12.金融业的发展不应高于实体经济。13.银行业融资和服务仍然是我国金融资源配置和发展的主要方式。14.中央和地方金融格局发生了巨大变化，解决了地方之间争夺资源、监管能力不够等问题。地方金融机构将受到严格审批，业务限制在本地，不许风险外溢，监管更加严格。15.国有金融机构在市场上占据主导地位，支持大型金融机构做优做强，成为服务实体经济的主力军和维护金融稳定的压舱石。同时严格控制中小金融机构的准入标准和监管要求，明确国有金融机构的主导地位。16.加快处置小金融机构的风险是政府的责任。政府需要注资、清理不合格股东、清理不良资产并注入优质资产等。中小银行改革化险给出了路线图。17.化解地方政府债务风险是必要的。通过展期、借新换旧、置换等方式稳妥化解存量债务，严格限制新增债务。中央政府已经下定决心解决这个问题，鼓励地方政府出售资产来偿还债务和隐形债务等。18.对地方融资平台进行分类管理是必要的。高债务的省份只能进行新债还旧的操作而不能新增债务。这有助于防范道德风险。19.在紧急情况下，央行可以通过紧急贷款为地方政府提供紧急流动性支持，但最长期限不得超过两年，仅用于“紧急事项”。20.对于城投债有特定的要求。地方政府，特别是县一级政府，将不再被允许发行新的债券来偿还旧的债券，需要出具说明文件来解释情况。21.继续进行金融反腐是必要的措施。对于互联网平台的金融业务没有明确的指示或规定。22.现在不再提及金融科技而是“科技金融”和“数字金融”。这包括引导和促进银行业、证券业、保险业等金融机构及创业投资等各类资本的运作和发展AI技术等措施。

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/ba1d251d-e1ff-42e3-983f-b1938642a4a7)

**Comment by Lewisrxliu** [comment]
USDT？
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/11e1d757-7302-4205-a83e-a34e2571ceea)

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/3e47128d-8de3-44fc-9f14-9d7ebd806eba)

**Comment by Lewisrxliu** [comment]
2024继续解读

![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/b175b8a8-f40e-4e6c-bd4e-1ae6e11afe78)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/daebf75e-dcc8-4a7a-a2fc-1c2c97869456)

**Reply by Lewisrxliu** [reply]
统计局的2023数据靠谱吗？
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/aed5f43f-e3c3-4f64-b2f9-1abfa969f041)

**Comment by Lewisrxliu** [comment]
1.19.2024
https://www.businessinsider.com/china-economy-outlook-stagnation-xi-jinping-property-crisis-paul-krugman-2024-1

**Reply by Lewisrxliu** [reply]
新质生产力理论?
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/4fb66f81-5298-4db7-a2c9-d09b6b454ae1)
